cli/README.md

```diff
@@ -65,16 +64,27 @@ The following table includes short descriptions and the general syntax for all o
 
 | Command  | Description  |
 |:---:|---|
-| `add-resource`  | Adds a resource to a service within your project in the specified stage |
-| `auth`  | Authenticate the Keptn CLI against a Keptn installation  |
-| `create`  | Create currently allows to create a project |
+| `abort`  | Aborts the execution of a sequence |

```

Comment:
I might favor using the imperative for the verb in the description, so this would be "Abort the execution of a sequence".  It makes the prose a little tighter and saves a space, always good in a table.  The current form is not wrong, of course -- this is just my preference.

Reply:
Hi @StackScribe, I agree with using the imperative for the descriptions. Perhaps this is a change that can be made in another issue along with an adjustment to the `keptn help` command, as I've matched the descriptions in the README the command's output. Specifically, they match the following:

https://user-images.githubusercontent.com/42628024/164546056-6e626cb3-e0a1-4a7c-a4ce-8fd19b1051aa.png

Reply:
Sounds good!